dataplane/standalone/switch.cc

```diff
@@ -225,8 +227,10 @@ sai_status_t Switch::create(_In_ uint32_t attr_count,
       .value = {.u32 = 1024},
   });
 
-  auto trGrpOid =
-      this->attrMgr->create(SAI_OBJECT_TYPE_HOSTIF_TRAP_GROUP, this->id);
+  sai_object_id_t trGrpOid;
+  this->translator->create(SAI_OBJECT_TYPE_HOSTIF_TRAP_GROUP, &trGrpOid, 0,
+                           nullptr);
+
```

Comment:
Should the other statements call translator->create as well? Maybe have a TODO for that?

Reply:
added todo